conda-forge.yml

```diff
@@ -1,4 +1,13 @@
 channel_priority: strict
+provider:
```

Comment:
Please don't make any changes outside of the `recipes/$PKG_NAME` folder when making PRs to staged-recipes.

Reply:
DONE